front/src/applications/stdcm/hooks/useStdcm.ts

```diff
@@ -43,6 +43,7 @@ const useStdcm = () => {
     STDCM_REQUEST_STATUS.idle
   );
   const [pathProperties, setPathProperties] = useState<ManageTrainSchedulePathProperties>();
+  const [isStdcmResultsEmpty, setIsStdcmResultsEmpty] = useState<boolean | undefined>(undefined);
```

Comment:
Can't this be false by default ? If I understand this state correctly, it's used to insert a failed simulation (only if path not found) in the list so initializing it with false shouldn't be an issue.

```suggestion
  const [isStdcmResultsEmpty, setIsStdcmResultsEmpty] = useState(false);
```

Reply:
Done